code/mob/living/critter.dm

```diff
@@ -88,6 +88,9 @@ ABSTRACT_TYPE(/mob/living/critter)
 
 	var/pull_w_class = W_CLASS_SMALL
 
+	///If the mob has an ai, turn this to TRUE if you want it to fight back upon being attacked
+	var/ai_retaliates = FALSE
```

Comment:
why false by default?

Reply:
lul simultaneous review. Also I think it's better to be false by default, so peaceful critters don't suddenly become homicidal if you accidentally hit them. I'd say it's better to have the behaviour require explicit enabling.